src/app/screens/Home/index.tsx

```diff
@@ -286,6 +289,17 @@ function Home() {
 
   return (
     <div>
+      {allowance && (
+        <Header
+          title="Back to Home"
```

Comment:
how about we use `allowance.host` here? 

In the other screens we use the title showing the "name" of the current page. e.g. on the send screen it is "send".

Reply:
Ye I agree, the back button speaks for itself aswel.